src/Illuminate/Validation/Concerns/ValidatesAttributes.php

```diff
@@ -1543,6 +1543,22 @@ public function validateUrl($attribute, $value)
         return preg_match($pattern, $value) > 0;
     }
 
+    /**
+     * Validate that an attribute is a valid UUID.
+     *
+     * @param  string $attribute
+     * @param  mixed  $value
+     * @return bool
+     */
+    public function validateUuid($attribute, $value)
+    {
+        if (! is_string($value)) {
+            return false;
+        }
+
+        return preg_match('/^[\da-f]{8}-[\da-f]{4}-[\da-f]{4}-[\da-f]{4}-[\da-f]{12}$/iD', $value);
```

Comment:
`preg_match` doesn't return `bool`, it returns `int`:

```suggestion
        return preg_match('/^[\da-f]{8}-[\da-f]{4}-[\da-f]{4}-[\da-f]{4}-[\da-f]{12}$/iD', $value) === 1;
```

Reply:
Followed validateUrl condition https://github.com/laravel/framework/blob/8de373628021098c7b3771a9745b63b6a03a84a4/src/Illuminate/Validation/Concerns/ValidatesAttributes.php#L1543